src/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,15 @@
+.card,
+[class*="card--"] {
+  @include card;
```

Comment:
Remove flex when we reach the smartphone size `@include media($smartphone) {
  @include card;
}`